qa/rpc-tests/httpbasics.py

```diff
@@ -20,83 +20,109 @@
 except ImportError:
     import urlparse
 
-class HTTPBasicsTest (BitcoinTestFramework):        
+class HTTPBasicsTest (BitcoinTestFramework):
     def setup_nodes(self):
         return start_nodes(4, self.options.tmpdir, extra_args=[['-rpckeepalive=1'], ['-rpckeepalive=0'], [], []])
 
-    def run_test(self):        
-        
+    def run_test(self):
+
```

Comment:
A relict from one of my previous pulls which i try to clean out without big noise. :-)